addons/mrp/static/tests/tours/mrp_manual_consumption.js

```diff
@@ -40,3 +40,27 @@ registry.category("web_tour.tours").add('test_mrp_manual_consumption_02', {
         },
         ...stepUtils.saveForm(),
 ]});
+
+registry.category("web_tour.tours").add('test_mrp_manual_consumption_03', {
+    test: true,
+    steps: () => [
+        {
+            trigger: "button:has(input[name='Nuclear Workcenter'])",
+        },
+        {
+            extra_trigger: "button.active:has(input[name='Nuclear Workcenter'])",
+            trigger: "button:contains('Confirm')",
+        },
+        {
+            trigger: "button.btn-light:contains('Nuclear Workcenter')",
+        },
+        {
+            extra_trigger: ".o_control_panel_actions button.active:contains('Nuclear Workcenter')",
+            trigger: ".o_finished_product span:contains('Finish')",
+        },
+        {
+            extra_trigger: ".o_mrp_display_record.o_active",
+            trigger: ".o_mrp_record_line:not(.text-muted) span:contains('Component')",
+            run: () => {},
```

Comment:
It is the case starting from the v 18.0 if I remember correctly. Sadly, the way tours work change a little bit in almost every version :/
Also, instead of using `run: () => {}`, you can use `isCheck: true` instead. I almost forgot that exist but its purpose is to check the trigger's target is in the page without click on it which is exactly what you want here :)
```suggestion
            isCheck: true,
```